packages/block-editor/src/components/block-draggable/style.scss

```diff
@@ -45,3 +48,10 @@
 		font-size: $default-font-size;
 	}
 }
+
+.block-draggable-invalid-drag-token {
+	.block-editor-block-draggable-chip-fade .block-editor-block-draggable-chip {
+		background-color: $gray-600;
+		opacity: 0.8;
```

Comment:
I don't think opacity is making it clear enough (perhaps introducing confusion even). I suggest gray-700, without the opacity. 

![CleanShot 2023-12-15 at 17 35 20](https://github.com/WordPress/gutenberg/assets/1813435/b960d8ab-58ea-4529-b747-a81a616d10ce)